student.library/src/main/java/tds/student/sql/abstractions/ExamRepository.java

```diff
@@ -11,4 +14,5 @@
  */
 public interface ExamRepository {
   Response<Exam> openExam(final OpenExamRequest openExamRequest) throws ReturnStatusException;
+  void approveAccommodations(final UUID examId, final ApproveAccommodationsRequest approveAccommodationsRequest) throws ReturnStatusException;
```

Comment:
Can we add JavaDocs to the new method?  I created one for the open exam on the feature branch